lumicks/pylake/tests/test_correlated_stack.py

```diff
@@ -340,20 +340,26 @@ def test_export(rgb_tiff_file, rgb_tiff_file_multi, gray_tiff_file, gray_tiff_fi
     from os import stat
 
     for filename in (rgb_tiff_file, rgb_tiff_file_multi, gray_tiff_file, gray_tiff_file_multi):
-        savename = str(filename.new(purebasename=f"out_{filename.purebasename}"))
-        stack = CorrelatedStack(str(filename))
-        stack.export_tiff(savename)
-        stack.src._src.close()
-        assert stat(savename).st_size > 0
-
-        with tifffile.TiffFile(str(filename)) as tif0, tifffile.TiffFile(savename) as tif:
-            assert len(tif0.pages) == len(tif.pages)
-            assert tif0.pages[0].software != tif.pages[0].software
-            assert "pylake" in tif.pages[0].software
-            assert "Applied channel 0 alignment" in tif.pages[0].description
-            assert "Channel 0 alignment" not in tif.pages[0].description
-            for page0, page in zip(tif0.pages, tif.pages):
-                assert page0.tags["DateTime"].value == page.tags["DateTime"].value
+        for align in (True, False):
```

Comment:
Maybe we can avoid the nested loops by using [`itertools.product`](https://docs.python.org/3/library/itertools.html#itertools.product)

```python
for filename, align in itertools.product((rgb_tiff_file, rgb_tiff_file_multi, gray_tiff_file, gray_tiff_file_multi), (True, False)):
```